src/main/java/uk/gov/mca/beacons/service/model/BeaconType.java

```diff
@@ -0,0 +1,10 @@
+package uk.gov.mca.beacons.service.model;
+
+/**
+ * Enumeration of the possible beacon types.
+ */
+public enum BeaconType {
```

Comment:
@zackads @stuart-madetech This will need modelling in the frontend as well before we send the JSON to the API when we try to register a set of beacons